src/modules/fw_att_control/FixedwingAttitudeControl.cpp

```diff
@@ -531,19 +536,21 @@ void FixedwingAttitudeControl::Run()
 						    && ((now - pid_autotune.timestamp) < 1_s)) {
 
 							bodyrate_ff = matrix::Vector3f(pid_autotune.rate_sp);
+							rates_setpoint = rates_setpoint + bodyrate_ff;
 						}
 					}
 
 					/* Run attitude RATE controllers which need the desired attitudes from above, add trim */
-					float roll_u = _roll_ctrl.control_euler_rate(dt, control_input, bodyrate_ff(0));
+					const Vector3f att_control = _rate_control.update(rates, rates_setpoint, angular_accel, dt, _landed);
+					float roll_u = att_control(0) * _airspeed_scaling * _airspeed_scaling;
```

Comment:
The FF should be scaled with 1/TAS and the PI with 1/IAS^2. https://docs.px4.io/v1.12/en/flight_stack/controller_diagrams.html#airspeed-scaling

Reply:
Does this mean that rate controller essentially needs to have airspeed included in the controller calculation and we cannot unify the rate controller with multicopters? I was hoping that the airspeed scaling can be handed off to the allocation, and the rate controller only outputs "desired moments"

Reply:
airspeed scaling effects each of the gains in a slightly different way (for units), right? could we externalize the CAS vs TAS calculations outside and set the scaler on an individual basis on each signal? Other option -- have no concept of "scaler" on the signals, and just input an adapted (scaled) gain from outside. wdyt? would be good to generalize this stuff as much as possible. think e.g. if we have other scaling factors later.. do we need a separate consideration for every single one in the rate controller itself? or can we just say one of the inputs to the control method is the gains, and we handle appropriately scaling those gains for whatever specific case externally.

Reply:
Actually now i understand what @Jaeyoung-Lim was talking about (sorry - read through too quick) - I agree - why can't we put airspeed scaling on the actuators themselves in allocation? looking at the moment that's commanded, and then sussing out per our flight condition how much moment we actually expect to generate with a given actuator.

Reply:
Honestly I would start by finding out how much of an effect the different scaling on FF has - would volunteer to fly a couple of planes with no distinction between FF and P/I scaling. 
And if we need to keep it then the allocation seems to be the right place for it, yes.

Reply:
addressed with d3d1e43ab03a257ca2ad03a1c2c977a28813f5c7